lms/djangoapps/discussion_api/urls.py

```diff
@@ -17,7 +17,7 @@
         CourseView.as_view(),
         name="discussion_course"
     ),
-    url(r"^v1/users/{}".format(settings.USERNAME_PATTERN), RetireUserView.as_view(), name="retire_discussion_user"),
+    url(r"^v1/accounts/retire_forum", RetireUserView.as_view(), name="retire_discussion_user"),
```

Comment:
Was the previous `v1/users/{}` url correct? Or an error?

Reply:
That was an error, this updated one matches the REST API Endpoints doc.